internal/docker/option.go

```diff
@@ -0,0 +1,28 @@
+package docker
+
+//GetOpts - iterate the inbound Options and return a struct.
+func GetOpts(opt ...Option) Options {
+	opts := getDefaultOptions()
+	for _, o := range opt {
+		o(&opts)
+	}
+	return opts
+}
+
+// Option - how Options are passed as arguments.
+type Option func(*Options)
+
+// Options - how Options are represented.
+type Options struct {
+	withContainerImage string
+}
+
+func getDefaultOptions() Options {
+	return Options{}
```

Comment:
Similar question as above to defaulting the container image to `postgres` this will allow you to do: 
```
if err := c.CreateDevDatabase(c.Context, opts...); err != nil {...}
```

instead of needing to append an option like:
```
opts = append(opts, base.WithContainerImage("postgres"))
if err := c.CreateDevDatabase(c.Context, opts...); err != nil {....}
```